src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -402,42 +402,15 @@ void DoPermute(int start, int end)
         }
 
         private static async Task TestInProcAndRemoteWorkspace(
-            bool syncWithRemoteServer, params ImmutableArray<(string hintName, SourceText text)>[] values)
+            params ImmutableArray<(string hintName, SourceText text)>[] values)
         {
             // Try every permutation of these values.
             foreach (var permutation in Permute(values))
-            {
-                var gotException = false;
-                try
-                {
-                    await TestInProcAndRemoteWorkspaceWorker(syncWithRemoteServer, permutation);
-                }
-                catch (XunitException)
-                {
-                    gotException = true;
-                }
-
-                // If we're syncing to the remove server, we should get no exceptions, since the data should be matched
-                // between both.  If we're not syncing, we should see a failure since the two processes will disagree on
-                // the contents.
-                Assert.Equal(syncWithRemoteServer, !gotException);
-            }
-        }
```

Comment:
we had tests actually vvalidating that without this option that our host/oop get out of sync.  since hte option is gone and both most be in sync, all the tests validate that.